lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -116,21 +117,19 @@ def __init__(self, module, function=None):
                         }
 
     def define_protocol(self):
-        ''' Set protocol based on use_https parameters '''
+        """ Set protocol based on use_https parameters """
```

Comment:
You don't need to have spaces at the start and the end, but you need to have a trailing comma.